### PR TITLE
web/console: added unit tests

### PR DESCRIPTION
**Pull request by AnnaTsybel**

### CODE REVIEW

File: web/console/jestSetup.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const WebSocket = require('ws');
+global.WebSocket = WebSocket;
+
+customGlobal.fetch = require('jest-fetch-mock');
+customGlobal.fetchMock = customGlobal.fetch;
+
+// Disallow warnings and errors from console.
+customGlobal.console.warn = (message) => { throw new Error(message); };
+customGlobal.console.error = (message) => { throw new Error(message); };
```
Comment (TarasPotsikaylo) [inline web/console/jestSetup.ts]: add empty line everywhere

File: web/console/package.json [hunk 2]
```diff
@@ -12,27 +13,30 @@
         "chart.js": "3.4.1",
         "copy-webpack-plugin": "^9.0.0",
         "ethers": "5.5.1",
+        "jest": "26.6.0",
+        "jest-transform-stub": "2.0.0",
```
Comment (TarasPotsikaylo) [inline web/console/package.json:17]: I think, this should be in devDependencies

File: web/console/package.json [hunk 3]
```diff
@@ -12,27 +13,30 @@
...
+        "react-redux": "7.2.9",
         "react-router-dom": "5.2.0",
         "react-scripts": "4.0.3",
         "react-slick": "0.28.1",
         "react-toastify": "8.0.3",
-        "react-unity-webgl": "^7.1.10",
-        "redux": "4.1.0",
-        "redux-thunk": "2.3.0",
+        "redux": "4.1.2",
+        "redux-mock-store": "1.5.4",
```
Comment (TarasPotsikaylo) [inline web/console/package.json]: and this

File: web/console/package.json [hunk 4]
```diff
@@ -12,27 +13,30 @@
...
+        "redux-thunk": "2.4.1",
         "slick-carousel": "1.8.1",
-        "stylelint-scss": "3.21.0"
+        "stylelint-scss": "3.21.0",
+        "ts-jest": "26.4.1"
```
Comment (TarasPotsikaylo) [inline web/console/package.json]: and this

File: web/console/src/app/views/CardPage/index.tsx [hunk 5]
```diff
@@ -48,7 +48,7 @@ const Card: React.FC = () => {
         }
     }
 
-    const mint = async () => {
+    const mint = async() => {
         const walletService = new WalletService(user);
         await walletService.mintNft(id);
```
Comment (TarasPotsikaylo) [inline web/console/src/app/views/CardPage/index.tsx:53]: add try catch

File: web/console/src/tests/unit/lootbox.spec.ts [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+const useDispatchMock = jest.spyOn(reactRedux, "useDispatch");
+const useSelectorMock = jest.spyOn(reactRedux, "useSelector");
+let updatedStore: any = mockStore(initialState);
+const mockDispatch = jest.fn();
+useDispatchMock.mockReturnValue(mockDispatch);
+updatedStore.dispatch = mockDispatch;
+
+
+
+
```
Comment (TarasPotsikaylo) [inline web/console/src/tests/unit/lootbox.spec.ts]: to many lines

File: web/console/src/tests/unit/user.spec.ts [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+        "0202def02b66279e3c4a93f484716077ae0196d78bb6e681898785bd7faceb9f7749",
+        "test@test.com",
+        "00000000-0000-0000-0000-000000000000",
+        "2021-12-17T00:31:52.437252Z",
+        "",
+        "2022-12-17T00:31:51.874508Z",
+        "0x0000000000000000000000000000000000000000",
+        "casper-wallet",
+    )
+    ;
```
Comment (TarasPotsikaylo) [inline web/console/src/tests/unit/user.spec.ts]: replace

File: web/console/src/app/views/CardPage/index.tsx (hunk 5 above)
Comment (AnnaTsybel) [reply web/console/src/app/views/CardPage/index.tsx:53]: Try catch is in casper/velas/ ethers services, there are notifications about errors too

**Comment by TarasPotsikaylo** [comment]
**For feature:** 
pleace, separate PR: one test - one PR

File: web/console/src/tests/unit/user.spec.ts (hunk 7 above)
Comment (AnnaTsybel) [reply web/console/src/tests/unit/user.spec.ts]: where? above initial state?????

Comment (TarasPotsikaylo) [reply web/console/src/tests/unit/user.spec.ts]: you have a semicolon on a separate line

**Review by TarasPotsikaylo — APPROVED** [review]

**Pull request merged**